pints/_log_likelihoods.py

```diff
@@ -117,6 +117,120 @@ def __call__(self, x):
                       - np.sum(autocorr_error**2, axis=0) / (2 * sigma**2))
 
 
+class GaussianIntegratedUniformLogLikelihood(pints.ProblemLogLikelihood):
```

Comment:
There's no further reference to the "Uniform" bit in the name. Is there also a non-uniform version of this likelihood?

Reply:
There will be later on, yes. In the docstrings I refer to why the uniform part of the name is needed.